share/spack/gitlab/cloud_pipelines/stacks/data-vis-sdk/spack.yaml

```diff
@@ -0,0 +1,52 @@
+spack:
+  view: false
+  concretization: separately
+
+  config:
+    install_tree:
+      root: /home/software/spack
+      padded_length: 512
+      projections:
+        all: '{architecture}/{compiler.name}-{compiler.version}/{name}-{version}-{hash}'
+
+  # Currently only hdf5 is inebaled to reduce the load on the CI build
+  # cluster.  The spec will be gradually expanded to enable all the SDK
+  # components.
+  specs:
+  - matrix:
+    - - ecp-data-vis-sdk~adios2~darshan~faodel+hdf5~pnetcdf~unifyfs~veloc~ascent~paraview~sz~vtkm~zfp
+
+  mirrors: { "mirror": "s3://spack-binaries-develop/data-vis-sdk" }
+
+  gitlab-ci:
+    image: { "name": "ghcr.io/scottwittenburg/ecpe4s-ubuntu18.04-runner-x86_64:2020-09-01", "entrypoint": [""] }
+    script:
+      - . "./share/spack/setup-env.sh"
+      - spack --version
+      - cd share/spack/gitlab/cloud_pipelines/stacks/data-vis-sdk
```

Comment:
This now needs to activate the concretized environment passed as artifact from the pipeline generation job.  So:

```
cd ${SPACK_CONCRETE_ENV_DIR}
```

Similar to the E4S one [here](https://github.com/spack/spack/blob/develop/share/spack/gitlab/cloud_pipelines/stacks/e4s/spack.yaml#L124).

Reply:
After you rebase, that is...

Reply:
Then I can empty your PR mirror if we want to see everything rebuild again.